src/components/common/dashboard_contents/pie-chart_parser.js

```diff
@@ -0,0 +1,80 @@
+export default function PieChartParser(arrayOfData) {

```

Comment:
Great job working on this! I'm worried that this will not scale, but I suppose that is a problem to solve in the future. :)